packages/core/scripts/postunlink/README.md

```diff
@@ -0,0 +1,9 @@
+# Uninstall Script
+
+## Universal Uninstall Script
+
+```shell
+node node_modules/@embrace-io/react-native/lib/scripts/postunlink/index.js
+```
+
+This script call `postunlink.android` and `postunlink.ios` to revert all changes that we made in the install script.
```

Comment:
could we use `uninstall` for the directory and all file names? Why was postunlink chosen?

Reply:
I can rename it and remove post unlink